frontend/test/__support__/e2e/cypress.js

```diff
@@ -328,3 +328,7 @@ export function generateUsers(count, groupIds) {
 
   return users;
 }
+
+export function enableSharingQuestion(id) {
```

Comment:
Nitpick: is this being used anywhere (yet)?

Reply:
@ariya Thanks for the catch, updated removing it.